src/AdminMenus/FormAdminMenu.php

```diff
@@ -426,18 +477,91 @@ private function getTopBarItems(string $type, string $formId): array
+						'additionalClass' => $bulkSelector,
+						'submitAttrs' => [
+							UtilsHelper::getStateAttribute('bulkType') => 'delete-entry',
+						],
+					]),
+					Components::render('submit', [
+						'submitVariant' => 'ghost',
+						'submitValue' => \__('Duplicate', 'eightshift-forms'),
+						'submitIsDisabled' => true,
+						'additionalClass' => $bulkSelector,
+						'submitAttrs' => [
+							UtilsHelper::getStateAttribute('bulkType') => 'duplicate-entry',
+						],
+					]),
 				];
 
 				$right = [
 					Components::render('submit', [
 						'submitVariant' => 'ghost',
-						'submitValue' => \__('Restore', 'eightshift-forms'),
+						'submitValue' => \__('Export to CSV', 'eightshift-forms'),
 						'submitIsDisabled' => true,
-						'additionalClass' => $bulkSelector,
+						'additionalClass' => "{$exportSelector} {$bulkSelector}",
 						'submitAttrs' => [
-							UtilsHelper::getStateAttribute('bulkType') => 'restore',
+							UtilsHelper::getStateAttribute('bulkType') => 'fake',
+							UtilsHelper::getStateAttribute('formId') => $formId,
 						],
 					]),
+				];
+				break;
+			case UtilsConfig::SLUG_ADMIN_LISTING_TRASH:
+				if ($parent === UtilsConfig::SLUG_ADMIN_LISTING_RESULTS) {
+					$left = [
+						Components::render('checkbox', [
+							'checkboxValue' => 'all',
+							'checkboxName' => 'all',
+							'additionalClass' => $selectAllSelector,
+						]),
+						Components::render('submit', [
+							'submitVariant' => 'ghost',
+							'submitButtonAsLink' => true,
+							'submitButtonAsLinkUrl' => UtilsGeneralHelper::getListingPageUrl(UtilsConfig::SLUG_ADMIN_LISTING_RESULTS),
+							'submitValue' => \__('Back', 'eightshift-forms'),
+							'submitIcon' => UtilsHelper::getUtilsIcons('arrowLeft')
+						]),
+						Components::render('submit', [
+							'submitVariant' => 'ghost',
+							'submitValue' => \__('Restore', 'eightshift-forms'),
+							'submitIsDisabled' => true,
+							'additionalClass' => $bulkSelector,
+							'submitAttrs' => [
+								UtilsHelper::getStateAttribute('bulkType') => 'restore',
+							],
+						]),
+					];
+				} else {
+					$left = [
+						Components::render('checkbox', [
+							'checkboxValue' => 'all',
+							'checkboxName' => 'all',
+							'additionalClass' => $selectAllSelector,
+						]),
+						Components::render('submit', [
+							'submitVariant' => 'ghost',
+							'submitButtonAsLink' => true,
+							'submitButtonAsLinkUrl' => UtilsGeneralHelper::getListingPageUrl(),
+							'submitValue' => \__('Back', 'eightshift-forms'),
+							'submitIcon' => UtilsHelper::getUtilsIcons('arrowLeft')
+						]),
+						Components::render('submit', [
+							'submitVariant' => 'ghost',
+							'submitValue' => \__('Restore', 'eightshift-forms'),
+							'submitIsDisabled' => true,
+							'additionalClass' => $bulkSelector,
+							'submitAttrs' => [
+								UtilsHelper::getStateAttribute('bulkType') => 'restore',
+							],
+						]),
+					];
+				}
```

Comment:
```suggestion
$submitButtonAsLinkUrl = $parent === UtilsConfig::SLUG_ADMIN_LISTING_RESULTS ? UtilsGeneralHelper::getListingPageUrl(UtilsConfig::SLUG_ADMIN_LISTING_RESULTS) : UtilsGeneralHelper::getListingPageUrl();

$left = [
    Components::render('checkbox', [
        'checkboxValue' => 'all',
        'checkboxName' => 'all',
        'additionalClass' => $selectAllSelector,
    ]),
    Components::render('submit', [
        'submitVariant' => 'ghost',
        'submitButtonAsLink' => true,
        'submitButtonAsLinkUrl' => $submitButtonAsLinkUrl,
        'submitValue' => \__('Back', 'eightshift-forms'),
        'submitIcon' => UtilsHelper::getUtilsIcons('arrowLeft')
    ]),
    Components::render('submit', [
        'submitVariant' => 'ghost',
        'submitValue' => \__('Restore', 'eightshift-forms'),
        'submitIsDisabled' => true,
        'additionalClass' => $bulkSelector,
        'submitAttrs' => [
            UtilsHelper::getStateAttribute('bulkType') => 'restore',
        ],
    ]),
]
```
Indenting in the provided suggestion may be off

Reply:
Just the `submitButtonAsLinkUrl ` is different in both `if` & `else` 🤷🏼

Reply:
future proffing